modules/campaignion_conf/campaignion_conf.features.wysiwyg.inc

```diff
@@ -52,7 +52,7 @@ function campaignion_conf_wysiwyg_default_profiles() {
       'acf_allowed_content' => '',
       'block_formats' => 'p,address,pre,h2,h3,h4,h5,h6,div',
       'css_setting' => 'theme',
-      'css_theme' => 'wyiswyg_theme_default',
+      'css_theme' => '',
```

Comment:
No, it should definitely *not* use the admin theme (that would be `wysiwyg_theme_admin` btw).

The WYSIWYG should be as true as possible to the actual page display which is achieved by using the default theme. At least that’s the decision we made a few years ago, I'm not aware this has changed?

Reply:
I’m not aware of that decision. For me it was a “this works the other thing doesn’t”. Do we guarantee that all the default theme’s styles work inside the wysiwyg editor? :grimacing:

Reply:
Yes, we usually do. And this change in the starterkit is not going to affect older custom themes, where this might not have been the case, anyway.

What makes you say “this works the other thing doesn’t”, do you have an example?

Reply:
> No, it should definitely *not* use the admin theme (that would be `wysiwyg_theme_admin` btw).

This setting would use the current drupal theme. On node edit pages this would be `ae_admin` — at least that’s what I think this setting does.

Reply:
`wysiwyg_theme_default` uses the site’s default theme
`wysiwyg_theme_admin` uses the admin theme
`wysiwyg_theme_active` uses the currently active theme